test/test_proxyweb.py

```
@@ -2595,6 +2318,411 @@ def test_clear_query_history_malformed_json(self):
+            # Wait for update to replicate
+            updated = False
+            for _ in range(10):
+                out = self._psql("postgres2", "testdb_pg2", "pguser2",
+                                 f"SELECT val FROM items_pg WHERE name = '{tag}'")
+                if "999" in out:
+                    updated = True
+                    break
+                time.sleep(0.5)
+
+            self.assertTrue(updated,
+                            f"Updated value did not replicate to subscriber within 5 seconds")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)